src/fairseq2/recipes/lm/instruction_finetune.py

```diff
@@ -138,7 +138,7 @@ class InstructionFinetuneConfig:
     """The step interval at which to checkpoint."""
 
     keep_last_n_checkpoints: Optional[int] = 1
-    """The number of checkpoints to keep. If ``None``, none will be deleted."""
```

Comment:
If the `None` behaviour is no longer supported, maybe we can restrict the type to be just int ?

Reply:
Oh, let me fix the docstr. The behavior has not changed. I was updating the docstrs for the upcoming training recipe and apparently I accidentally removed the second sentence.